pipe_satellite_data/utils/locations.py

```diff
@@ -103,19 +104,23 @@ def satellite_locations(tles, dt):
     end_ts = start_ts + SECONDS_IN_DAY
 
     for tle in tles:
-        tle_lines = [str(tle['TLE_LINE%s' % i]) for i in range(3)]
-        orbit = ephem.readtle(*tle_lines)
-
-        for ts in range(start_ts, end_ts):
-            orbit.compute(datetime.utcfromtimestamp(ts).strftime("%Y/%m/%d %H:%M:%S"))
-            lon = ephem.degrees(orbit.sublong) * 180 / 3.1416
-            lat = ephem.degrees(orbit.sublat) * 180 / 3.1416
-            elevation = orbit.elevation
-
-            yield dict(
-                norad_id=tle['NORAD_CAT_ID'],
-                lat=lat,
-                lon=lon,
-                timestamp=ts,
-                altitude=elevation
-            )
+        try:
+            tle_lines = [str(tle['TLE_LINE%s' % i]) for i in range(3)]
+            orbit = ephem.readtle(*tle_lines)
+
+            for ts in range(start_ts, end_ts):
+                orbit.compute(datetime.utcfromtimestamp(ts).strftime("%Y/%m/%d %H:%M:%S"))
+                lon = ephem.degrees(orbit.sublong) * 180 / 3.1416
+                lat = ephem.degrees(orbit.sublat) * 180 / 3.1416
+                elevation = orbit.elevation
+
+                yield dict(
+                    norad_id=tle['NORAD_CAT_ID'],
+                    lat=lat,
+                    lon=lon,
+                    timestamp=ts,
+                    altitude=elevation
+                )
+        except RuntimeError as e:
+            logging.error(e, f'Error found in TLE: {tle}.\nCheck satellite status: https://www.n2yo.com/satellite/?s={tle["NORAD_CAT_ID"]}')
+            raise e
```

Comment:
@bitsofbits we want to know that there is an error if something rises. This will let us know that the task failed in airflow side. Just did the catch to make it easy for detecting the TLE involved. do you agree?